src/app/components/Byline/utilities/bylineExtractor/index.tsx

```diff
@@ -26,47 +26,61 @@ const pathOrZeroIndexModelBlocks = (
 };
 
 const bylineExtractor = (blocks: OptimoBylineContributorBlock[]) => {
-  const bylineBlocks = blocks?.[0]?.model?.blocks || [];
+  return blocks
+    .slice(0, 4)
```

Comment:
Do we have an example of an article where there's 4 contributors. Should we try to verify what that looks like at the various different breakpoints? (I am concerned that 4 might not fit nicely, in which case we would need to reduce this to 2 or 3 until the byline revamp occurs)

Reply:
Me and Wendy have verified what it looks like with 4 contributors during the design phase (just by duplicating a contributor element), and it all fits nicely !